ui-lib/src/main/java/co/electriccoin/zcash/ui/Navigation.kt

```diff
@@ -280,6 +251,65 @@ internal fun MainActivity.Navigation() {
     }
 }
 
+/**
+ * This is the Home screens sub-navigation. We could consider creating a separate sub-navigation graph.
+ */
+@Composable
+private fun MainActivity.NavigationHome(
```

Comment:
This part was mainly moved from the `Navigation` to a separate Composable, although its bottom part contains necessary changes.